server/services/model.ts

```diff
@@ -100,11 +101,11 @@ export async function deleteModel(user: UserDoc, modelId: ModelId) {
 export async function removeVersionFromModel(user: UserDoc, version: VersionDoc) {
   // Deletes model if no versions left
   const model = await ModelModel.findById(version.model)
-
   if (!model) {
     throw NotFound({ modelId: version.model }, 'Unable to find model to remove.')
   }
 
+  // Remove model record
```

Comment:
I don't think this comment is correct. It's removing the version of a model, not the model itself.

I'd also argue that the comment isn't needed as the function name describes this clearly, but I don't feel too strongly about that.